### PR TITLE
get products by ownerId

### PR DESCRIPTION
**Pull request by matthewaustinbell**

### CODE REVIEW

File: StudentLoans/DataAccess/ProductRepository.cs [hunk 1]
```diff
@@ -78,6 +78,20 @@ public IEnumerable<Product> GetProductsByName(string searchTerm)
             }
         }
 
+
+        public IEnumerable<Product> GetSellersAvailableProducts(int ownerId)
+        {
+            using (var db = new SqlConnection(_connectionString))
+            {
+                var sql = @"select OwnerId, [Name], IsRented from Products
```
Comment (HeathJHMoore) [inline StudentLoans/DataAccess/ProductRepository.cs]: This looks good but the only other thing we need to do is add the rest of the product fields to the SQL Select statement
We could probably just get away with changing the query to this
```
SELECT p.*
 FROM Products p
 WHERE OwnerId = @OwnerId AND isRented = 0;
```

**Review by HeathJHMoore — CHANGES_REQUESTED** [review]

File: StudentLoans/DataAccess/ProductRepository.cs (hunk 1 above)
Comment (matthewaustinbell) [reply StudentLoans/DataAccess/ProductRepository.cs]: ok thanks i pushed that up if you would like to test it on your end

**Comment by HeathJHMoore** [comment]
👍

**Comment by HeathJHMoore** [comment]
👍

**Pull request merged**